package.json

```diff
@@ -3,15 +3,14 @@
   "nicename": "Blueprint Markdown Renderer",
   "version": "0.3.0",
   "engines": {
-    "node": "0.10.x",
-    "npm": "2.15.x"
+    "node": "0.12.x",
```

Comment:
since we are testing and trying to have it compatible for 0.10, 0.12 and at least 4.6, then I would specify version as: `>=0.10 <=4.6`